Makefile

```diff
@@ -3,36 +3,41 @@
 tools:
 	bash ./scripts/tools.sh
 
-build:
+prepare: generate-code
+
+generate-code:
+	bash ./scripts/generate-templates-api.sh
+
```

Comment:
Update kodu trva chvilku, pridal som ho pred vsetky prikazy, aby sa na to nemuselo mysliet.